media-nep/src/test/java/nl/vpro/nep/service/impl/NEPItemizeServiceImplITest.java

```diff
@@ -63,11 +63,20 @@ public void itemize() throws IOException, NEPException {
     @Test
     @Order(10)
     @Tag("dvr")
-    public void itemizeDvr() throws NEPException {
+    public void itemizeDvr() throws NEPException, InterruptedException {
         Instant start = Instant.now();
         NEPItemizeServiceImpl itemizer = new NEPItemizeServiceImpl(NEPTest.PROPERTIES);
         response = itemizer.itemizeLive("npo-1dvr", Instant.now().minusSeconds(300), Instant.now().minusSeconds(60), null);
         log.info("response: {} {}", response, start);
+        while(true) {
```

Comment:
In theorie van het hier eeuwig vastlopen.

Reply:
Het is een test die niet automatisch draait. GEbruikt om het even handmatig te bekijken. Maar ik heb een timeout op de hele test gezet.